source/docs/one-command-deployment.md

```diff
@@ -43,18 +47,17 @@ deploy:
   message: [message] #leave this blank
 ```
 
-| Option    | Description                                                                                                 |
-| --------- | ----------------------------------------------------------------------------------------------------------- |
-| `repo`    | GitHub/Bitbucket/Coding/GitLab repository URL                                                               |
-| `branch`  | Branch name. The deployer will detect the branch automatically if you are using GitHub or GitCafe.          |
-| `message` | Customize commit message (Default to `Site updated: {% raw %}{{ now('YYYY-MM-DD HH:mm:ss') }}{% endraw %}`) |
-
-3. Upload your site: `./node_modules/.bin/hexo clean && ./node_modules/.bin/hexo deploy` (or `hexo clean && hexo deploy` if you installed Hexo globally).
-4. On Github/BitBucket/Gitlab go to your repository settings and change your main branch from `master` to `published` (or whatever you called it in your \_config.yml). Now your site will show as your account's homepage.
-
-### How does it work exactly?
+| Option    | Description |
+| --------- | --- |
+| `repo`    | URL of the target repository |
+| `branch`  | Branch name. Defaults to `gh-pages` on GitHub and `master` for others |
+| `message` | Customize commit message. Defaults to `Site updated: {% raw %}{{ now('YYYY-MM-DD HH:mm:ss') }}{% endraw %}` |
+| `token`   | Optional token value to authenticate with the repo. Prefix with `$` to read token from environment variable |
 
-Your repository will have a **master** branch when you first made it. Keep working on this branch to create your site. When you deploy Hexo will create, or update, a new branch on the remote site (called **published** in the config above). Deployment won't create a new branch locally, nor will it mess with your existing source code in the master branch locally or on the remote. You still need to keep pushing commits to the master branch manually to the remote server to keep your site backed up. In addition, if you are using a CNAME file to customize your Github Pages domain name, you need to put the CNAME file under `source_dir` so that Hexo can push it to the published branch.
+3. Deploy your site, `./node_modules/.bin/hexo clean && ./node_modules/.bin/hexo deploy` (or `hexo clean && hexo deploy` if you installed Hexo globally).
+  - You will be prompted with username and password of the target repository, unless you authenticate with a token.
```

Comment:
not familiar with hexo-deployer-git, is this sentence correct?

Reply:
Usually we setup hexo-deployer-git with https git url and it requires prompt. I don't know what will happen if I set up with ssh git url.

Reply:
Mentioned `ssh key'.